src/DropdownItem.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5;
+
+use InvalidArgumentException;
+use Stringable;
+use Yiisoft\Html\Html;
+use Yiisoft\Html\Tag\A;
+use Yiisoft\Html\Tag\Button;
+use Yiisoft\Html\Tag\Hr;
+use Yiisoft\Html\Tag\Li;
+use Yiisoft\Html\Tag\Span;
+
+final class DropdownItem
```

Comment:
Add private constructor. Seems, this class not means with default value.

Reply:
The class is final, what's the point of adding a private empty constructor?

Reply:
The point of deny create `DropdownItem` instance via constructor, user must use one of static methods to create `DropdownItem`.

Reply:
done.